connector/subgraph/src/ConvictionVoting.ts

```diff
@@ -13,6 +14,9 @@ import {
   getStakeHistoryEntity,
   getOrgAddress,
 } from './helpers'
+import { STATUS_CANCELLED, STATUS_EXECUTED } from './proposal-statuses'
```

Comment:
shouldn't we just import `ProposalStatus` enum from the schema and use what we have there?

Reply:
Don't think there's a way to import the declared enums from the schema. Do you know any?